src/System.Data.Common/tests/System/Data/Common/DataColumnMappingTest.cs

```diff
@@ -0,0 +1,70 @@
+using System.Data.Common;
+using Xunit;
+
+namespace System.Data.Tests.Common
+{
+    public class DataColumnMappingTest
+    {
+        [Fact]
+        public void DataSetColumn_Get_WhenNotSet()
+        {
+            DataColumnMapping dataColumnMapping = new DataColumnMapping();
+
+            string dataSetColumn = dataColumnMapping.DataSetColumn;
+
+            Assert.Equal("", dataSetColumn);
```

Comment:
I'm not sure we're consistent about that? If anything I think I see "" more.
In the past I used String.Empty consistently. I assume "" is interned by the compiler at least per assembly, I suppose String.Empty avoids that duplication, but it seems really marginal.